### ISSUE TITLE
[MySQL] Unsigned decimal types are not exported properly.

### ISSUE THREAD

**Issue by shubham-yb** [yugabyte/yb-voyager#573]
Unsigned decimal types lose their precision and have `UNSIGNED` exported along with them. As we know, unsigned is not supported by YB.

Source schema:

```
create table if not exists fixed_point_types( 

                                             d_us decimal(10,2) unsigned,
					     dec_type dec(5,5) unsigned,
                                             numeric_type numeric(10,5) unsigned,
                                             fixed_type fixed(10,3) unsigned
);
```

Exported schema:

```
CREATE TABLE fixed_point_types (
        d_us DECIMAL UNSIGNED,
        dec_type DECIMAL UNSIGNED,
        numeric_type DECIMAL UNSIGNED,
        fixed_type DECIMAL UNSIGNED
) ;
```

Suggested Workaround:
```
CREATE TABLE fixed_point_types (
        d_us DECIMAL(10,2),
        dec_type DECIMAL(5,5),
        numeric_type DECIMAL(10,5),
        fixed_type DECIMAL (10,3)
) ;
```

**Comment by shubham-yb** [comment]
Fixed by ora2pg in their commit `f08d47a4e5c20ff7d348e535ed05ab571b4bb350` and tested.

**Comment by shubham-yb** [comment]
The fix has landed in our main due to Ora2pg hash update in the installer script.
